src/figuresClasses.ts

```diff
@@ -1,11 +1,75 @@
-export interface Figure {}
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+
+  constructor(
+    public color: Color,
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Sides must have positive value');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Sides do not form a valid triangle');
+    }
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+
+    return (
+      Math.round(
+        Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c)) * 100,
+      ) / 100
```

Comment:
The rounding logic in the `getArea` method of the `Triangle` class could be simplified. Consider using `toFixed(2)` to round the result to two decimal places instead of manually multiplying and dividing by 100.